### ISSUE TITLE
pretrained weights prediction have a little bias?

### ISSUE THREAD

**Issue by jinfagang** [liuruijin17/LSTR#17]
Hi, do u know why does this bias happen?

![image](https://user-images.githubusercontent.com/21303438/101325575-f0c37500-38a6-11eb-916b-34804393a754.png)

**Comment by liuruijin17** [comment]
Hi,
I think the reasons may consist of (1) lack of appearance clues at the remote horizon; (2) lack of aforementioned data.

**Comment by jinfagang** [comment]
@liuruijin17 Did u notice these samples which happens mostly in curved scenarios? Is there a way to make LSTR solve it? (Otherwise this problem was quite same as UltraFfastLane algorithm which based on anchor, the lane line can not locate on lane mark tightly)

**Comment by liuruijin17** [comment]
Hi @jinfagang 
We have noticed these samples and here are some immature ideas:
* The number of labeling points at the remote horizon is very sparse.
* The data imbalance between points at the curved regions and linear aligned points.
* The curvature distribution of **point** is hard to balance, even though the number of curved **lanes** can be ensured to collect a sufficient number of training lanes.
* The appearance information at the remote horizon is severely compressed due to projective transformation.

How to solve it?
* New training loss which can balance the data imbalance.
* Novel architecture to focus on the remote region.
* More suitable mathematical modeling for the lane structures at the remote region.

**Comment by jinfagang** [comment]
@liuruijin17 Waiting for LSTRv2.0

**Comment by liuruijin17** [comment]
@jinfagang Thank you for your interest and support, and it is a pleasure to discuss this with you!
